### DISCUSSION TITLE
CoPilot and GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by mikgur** [community/community#43696] [GitHub Education] [answered]
Hello, I was approved to get access to GitHub Student Developer Pack in the late 2022. Since then I was using CoPilot, but today it stopped working. When I try to sign in - there is a standard subscription option ($10 per month).

Is CoPilot still in the GitHub Student Developer Pack? How can I re-enable it?

Thanks!

**Comment by RupertBothma** [comment]
Same

**Comment by jongwooo** [comment]
Solved: #43634

**Comment by iheboueslatiesprit** [comment]
same

**Comment by mikgur** [comment, marked as answer]
CoPilot just start working today without any email :-)

**Reply by J2TeamNNL** [reply]
Not for me

**Comment by lorentzen99** [comment]
I have the same issue

**Comment by standoge** [comment]
Same.
![image](https://user-images.githubusercontent.com/77755762/211968724-84612579-0456-40be-a5f2-fd7eb98690b3.png)

**Reply by standoge** [reply]
Today, it started working again without any action needed ✅

**Comment by cogsandsquigs** [comment]
Same here. Hopefully its just a bug :P
![image](https://user-images.githubusercontent.com/59457601/211969157-4d219a6e-ca85-44cb-b86d-5fd030bd01b3.png)

**Comment by tianyu-z** [comment]
So we just need to wait?

**Comment by stlukey** [comment]
**Just stopped working for me too. Wasn't affected by the error 2 days ago.** Suddenly, IDE reports I need to get a subscription and the website asks me to get a membership when clicking enable copilot.

**Comment by jiayichengcode** [comment]
Same here. Verified in July 2022, and reverify page said I do not need to do so now.

**Reply by 0xffffharry** [reply]
Me too

**Comment by 2sBs** [comment]
Have also lost Copilot access completely, after being rate-limited for the last few days.

**Reply by 2sBs** [reply]
Have lodged a support ticket also, will reply back if I hear anything

**Reply by 2sBs** [reply]
Haven't heard anything, but my Copilot is working again. From the GitHub Status page:
![image](https://user-images.githubusercontent.com/30145698/211979577-68d9e65c-8db2-4653-b5c0-1a0e1d2ceb11.png)

**Comment by Hellscaped** [comment]
same!

**Comment by gutenfries** [comment]
+1 same issue :(

forced me to realize how much I really use copilot for certain things...

**Comment by Arturossi** [comment]
Same!
The whole day copilot was unstable but was still working. Now it is down and asks for a license. :/

**Reply by MaddozS** [reply]
> Same! The whole day copilot was unstable but was still working. Now it is down and asks for a license. :/

I have the same issue, today was working but unstable, now not working at all

**Comment by aldev814** [comment]
Same!

**Comment by NorthPhoenix** [comment]
Same here!

**Comment by rusdianto05** [comment]
now not working again

**Comment by Hasan-Rifat** [comment]
issue 

![github](https://user-images.githubusercontent.com/80412774/211977588-0c965429-cb55-48a3-9116-06dd5477473d.png)

**Comment by Ayfri** [comment]
Same issue.

**Comment by mrmezan06** [comment]
![image](https://user-images.githubusercontent.com/43064536/213264306-7a42aabe-1d7c-41ea-80c0-2d6584fe9af9.png)

What should I do now?

**Reply by standoge** [reply]
If you are inside of Student Developer Pack, it will start to work again in some hours, don't worry.

**Reply by standoge** [reply]
It happens to me some days ago, but now it's working. https://github.com/orgs/community/discussions/43696#discussioncomment-4663150

**Comment by fnhirwa** [comment]
I faced the same issue🤔

**Comment by sylasop** [comment]
I have github student package, but suddenly it shows like this any fixes yet?
<img width="953" alt="Screenshot 2023-01-19 at 1 02 45 AM" src="https://user-images.githubusercontent.com/92511866/213273914-ba7c5c9e-f7e5-4eb9-9bf0-a01b30ddba7f.png">
